spec/API_specification/signatures/_types.py

```diff
@@ -33,7 +33,7 @@ class iinfo_object:
     min: int
 
 # This should really be recursive, but that isn't supported yet.
-NestedSequence = Sequence[Sequence[Any]]
+NestedSequence = Sequence[Any]
```

Comment:
Fyi: mypy has now supported nested protocols for a few releases, so it is now possible to express recursive sequences:
``` python
from __future__ import annotations

from typing import TypeVar, Protocol

_T_co = TypeVar("_T_co", covariant=True)

class NestedSequence(Protocol[_T_co]):
    def __getitem__(self, key: int, /) -> _T_co | NestedSequence[_T_co]: ...
    def __len__(self, /) -> int: ...
```

Reply:
Thanks for the info!